src/lib.rs

```diff
@@ -235,12 +236,9 @@ static mut LOGGER: *const Log = &NopLogger;
 static STATE: AtomicUsize = ATOMIC_USIZE_INIT;
 static REFCOUNT: AtomicUsize = ATOMIC_USIZE_INIT;
```

Comment:
No - because of shutdown, the 0.3.x internal implementation is basically the same as it was before. The 0.4.x logger we generate is a unit struct and just references the 0.3.x global state.